packages/utils/src/utils/index.js

```diff
@@ -113,7 +113,8 @@ export function getStatus(resource) {
 
 export function isRunning(reason, status) {
   return (
-    status === 'Unknown' && (reason === 'Running' || reason === 'Building')
+    !status ||
```

Comment:
Should we not differentiate between 'pending' and 'running'? Showing a running icon could be a bit misleading if a run is just queued but hasn't been applied yet.

Reply:
Sounds good, Mind pointing me towards a suitable carbon icon ?

Reply:
do we need an icon at all for this state? I'll have a look though, give me a while

Reply:
https://www.carbondesignsystem.com/guidelines/iconography/library/

Since we're already using the 'checkmark' and 'error' icons from the status block we might consider 'undefined' or 'unknown'
![image](https://user-images.githubusercontent.com/2829095/65151094-4ea73b00-da1d-11e9-9b67-6670ef7dc383.png)

However, I think 'time' might also work:
![image](https://user-images.githubusercontent.com/2829095/65151124-5ebf1a80-da1d-11e9-9eb1-8c129f9d8d69.png)